article1_fe/article1-fe/src/components/board/ContentArea.vue

```diff
@@ -10,11 +20,33 @@ const props = defineProps({
   }
 });
 
-const emit = defineEmits(["delete"]); // 삭제 이벤트 정의
+const authStore = useAuthStore();
+
+const route = useRoute();
+
+const emit = defineEmits(["delete"]);
 
 // 신고 버튼 클릭 이벤트
-function blameBoard() {
-  console.log("신고");
+async function blameBoard() {
+  try {
+    const boardSeq = route.params.boardSeq;
+    const response = await axios.post(`http://localhost:8080/blame/board/${boardSeq}`, {}, {
```

Comment:
baseUrl 설정해놔서, 추후에 아래와 같이 변경해주세요 !
```suggestion
const response = await axios.post(`/blame/board/${boardSeq}`, {}, {
```